package/tessel/tessel-version/files/version.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "0.0.1",
+  "commits": {
+    "openwrt": "8ec549f3d1a76647c24324a21c430c70cabf3b19",
+    "samd21": "cfe3cdb324bc78935077bb0507c9f7a846f1e2ae"
```

Comment:
A file in squashfs won't tell you what's on the SAMD21. Or is this the version that's supposed to be on it?

Or is it the commit that the installed copy of tessel.js, usbexecd and spid come from? You can already obtain that version from opkg.